### ISSUE TITLE
odd  copyright/license situation ?

### ISSUE THREAD

**Issue by gempesaw** [carandraug/pod-weaver-section-legal-complicated#1]
tl; dr -  Some of the files have explanations of the file's history ("Module W was authored by X, expanded by XX, inspired by Y, and sponsored by Z"). 

continued from carandraug/pod-weaver-section-generatesection#1:

I recently became the maintainer of two separate modules that were previously merged because they were related and some of the maintainers were no longer interested; I didn't want to delete the historical notes.

[Selenium::Remote::Driver](https://github.com/gempesaw/Selenium-Remote-Driver/tree/master/lib/Selenium/Remote) has a handful of files that are easy: apache2 license, old copyright owners, and now me as the new owner. I'm using [GenerateSection to do the copyright](https://github.com/gempesaw/Selenium-Remote-Driver/blob/a2a0fb9ec836eb818a61c40192e5d68eb6bda2ff/weaver.ini#L37-L52), although I think Legal::Complicated would work there.

[Test::Selenium::Remote::Driver](https://github.com/gempesaw/Selenium-Remote-Driver/tree/master/lib/Test/Selenium/Remote), the second module, has two files that are under the perl5 artistic license. One of them was authored by Luke Closs, and later maintained by Mark Stosberg, and sponsored by Prime Radiant, and has a separate contributors section. The other seems to have been authored entirely by Mark Stosberg? I'm not quite sure about the details of that, but basically the attribution sections author sections have extraneous text in them explaining the history of the files, and mentioning company sponsorships.

Complicating the issue was that I _did_ want the PodWeaver stuff to go in the S::R::D files, which had plenty of redundant POD, but ended up not needing any weaving for the Test::S::R::D ones.
